op/model.py

```diff
@@ -194,26 +207,34 @@ def __getitem__(self, key):
 
 # We mix in MutableMapping here to get some convenience implementations, but whether it's actually
 # mutable or not is controlled by the flag.
-class RelationUnitData(LazyMapping, MutableMapping):
-    def __init__(self, relation, unit, is_mutable, backend):
+class RelationEntityData(LazyMapping, MutableMapping):
+    def __init__(self, relation, entity, is_mutable, backend):
         self.relation = relation
-        self.unit = unit
+        self.entity = entity
         self._is_mutable = is_mutable
         self._backend = backend
 
+        if isinstance(self.entity, Application):
+            self._is_app = True
+        else:
+            self._is_app = False
```

Comment:
```
self._is_app = isinstance(entity, Application)
```

Reply:
Indeed, I wonder how I didn't notice this.